resource_customizations/apps/Deployment/actions/discovery.lua

```diff
@@ -7,4 +7,18 @@ if obj.spec.paused ~= nil then
     actions["pause"] = {paused}
 end
 actions["resume"] = {["disabled"] = not(paused)}
+
+actions["scale"] = {
+    ["params"] = {
+        {
+            ["name"] = "scale",
+            ["value"] = "",
+            ["type"] = "^[0-9]*$",
```

Comment:
nitpick on the name, but having a "type" be a validation regexp is not really intuitive.

```suggestion
            ["format"] = "^[0-9]*$",
```